src/main/resources/org/jenkins/plugins/statistics/gatherer/StatisticsConfiguration/config.groovy

```diff
@@ -60,5 +60,13 @@ f.section(title:_("Statistics Gatherer")) {
         f.entry(title:_("Enable HTTP publishing?"), field:"shouldSendApiHttpRequests") {
             f.checkbox(default: instance.shouldSendApiHttpRequests == null ? false : instance.shouldSendApiHttpRequests.equals(true));
         }
+
+        f.entry(title:_("Enable publish of events to LOGBack"), field:"shouldSendToLogback") {
+            f.checkbox(default: false)
```

Comment:
I'm not really aware of how `f.checkbox` works. It's been a while since I used it. In this case, won't it always be false? 

If not, maybe we should change the handling of the checkboxes in the rest of the class

Reply:
Let me check.